data/homePage.json

```diff
@@ -0,0 +1,83 @@
+    }
+  ],
+  "ecosystem": [
+    {
+      "title": "Access from anywhere",
+      "list": [
+        "homePageImg/ecosA1.png",
+        "homePageImg/ecosA2.png",
+        "homePageImg/ecosA3.png",
+        "homePageImg/ecosA4.png",
+        "homePageImg/ecosA5.png",
+        "homePageImg/ecosA6.png"
+      ]
+    },
+    {
+      "title": "Deploy at any scale",
+      "list": [
+        "homePageImg/ecosB1.png",
+        "homePageImg/ecosB2.png",
+        "homePageImg/ecosB3.png",
+        "homePageImg/ecosB4.png"
+      ]
+    },
+    {
+      "title": "Connect to any data",
+      "list": [
+        "homePageImg/ecosC1.png",
+        "homePageImg/ecosC2.png",
+        "homePageImg/ecosC3.png",
+        "homePageImg/ecosC4.png",
+        "homePageImg/ecosC5.png",
+        "homePageImg/ecosC6.png"
+      ]
+    }
+  ],
+  "useCases":[
+    {
+      "title":"Interactive Analytics",
+      "desc":"Kyuubi is an advanced, enterprise-grade, rapid analytics platform for interactive visual analytics on big data, with modern computing frameworks under the hood, i.e., Apache Spark, Apache Flink, Trino, e.t.c. With JDBC/ODBC, users can access kyuubi and run queries efficiently through SQL directly or generated by BI tools. Kyuubi caches background engine instances at the user level for better computing resource sharing and quick response. They parallelize queries on large amounts of data and return the results quickly."
+    },
+    {
+      "title":"Batch Processing",
+      "desc":"Kyuubi provides a SQL interface that you are already comfortable with for batch processing, typically large Extract, Transform, Load (ETL) processes. Both kyuubi and its engines are storage independent and work with numerous data sources, and Kyuubi isolates background engine instances at the connection level for better computing resource isolation and stability."
+    },
+    {
+      "title":"Data Lake & Lakehouses",
+      "desc":"Kyuubi supports query all traditional data warehouses, delta lakes, like  Apache Hive/HDFS, or modern lakehouses, like Apache Iceberg, Apache Hudi, and Delta Lake, together. Kyuubi also provides multi catalog meta APIs that present a sizeable centralized picture of all your data in front of you and help you innovate faster. The ability to query disparate data sources in a single entrance with the ANSI standard SQL syntaxes greatly simplifies data insight while providing authentication and authorization to keep all data secure."
+    }
+  ],
+  "community": [
+    {
+      "img": "homePageImg/community1.png",
+      "title": "Mailing list",
+      "href": "https://kyuubi.apache.org/mailing_lists.html"
+    },
+    {
+      "img": "homePageImg/community2.png",
+      "title": "Source code",
+      "href": "https://github.com/apache/incubator-kyuubi"
+    },
+    {
+      "img": "homePageImg/community3.png",
+      "title": "How to contribute",
+      "href": "https://kyuubi.apache.org/contributing.html"
```

Comment:
This link accesses 404, should it point to https://kyuubi.apache.org/issue_tracking.html ?

Reply:
@cxzl25 good catch !

Reply:
do you have time to fix it ? thank you !

Reply:
maybe we shall use https://github.com/apache/incubator-kyuubi/blob/master/CONTRIBUTING.md

Reply:
open a PR: https://github.com/apache/incubator-kyuubi-website/pull/52